### ISSUE TITLE
Incident chat room should support video chat, too

### ISSUE THREAD

**Issue by meitar** [betterangels/better-angels#165]
Although this is unlikely to be possible on shared hosting environments, more robust servers can support additional components (like an XMPP server). Buoy should be able to detect the presence of these additional servers (or should be able to accept a custom configuration from a Buoy Admin to use them) and then modify the incident chat room to support them. For example, if I have my own server (a VPS) and install the [Jitsi Videobridge](https://jitsi.org/videobridge/) on it, then I should be able to tell Buoy that this server exists and Buoy should make use of it in the incident chat room, similar to [Jitsi Meet](https://github.com/jitsi/jitsi-meet).

**Comment by meitar** [comment]
I took the easy route to this issue for now and used the Jitsi Meet External API to create a new chat room system for Buoy that uses the Jitsi Meet web service. This gives Buoy the appearance of built-in video conferencing, so long as the chat room option used is "Jitsi Meet."

I'd still like to work towards providing a self-hosted integration, but for now this is a quick way to make a big splash.

The TL;DR: Buoy version 0.2.0 will support Jitsi Meet and thus integrated video chat out of the box. :)
